src/configprovider-cppdbg.ts

```diff
@@ -0,0 +1,55 @@
+import * as vscode from "vscode";
+import { Target } from "./types";
+import { MesonDebugConfigurationProvider } from "./configprovider";
+
+export enum MIModes {
```

Comment:
Does this need to be exported anymore?

Reply:
I don't know. It was exported before and I don't know the language.

Reply:
The documentation for this is at https://code.visualstudio.com/docs/cpp/launch-json-reference#_mimode and it's just a string that can have two values set.

TBH, I see zero value in having this export here. It just makes you do more reading and writing, so it's actually a net negative. I will remove it and just set the raw strings where appropriate.

```ts
export enum MIModes {
  lldb = "lldb",
  gdb = "gdb",
}
```